root/static/scripts/edit/externalLinks.js

```diff
@@ -603,6 +603,13 @@ export class ExternalLinksEditor
                     Enter the destination URL instead.`),
         target: URLCleanup.ERROR_TARGETS.URL,
       };
+    } else if (isNewOrChangedLink && isGoogleSearch(link.url)) {
+      error = {
+        message: l(`Please don’t enter links to Google searches.
+                    If you’ve found any links through your search
+                    that seem useful, do enter those instead.`),
```

Comment:
Found at least one Bing search using http://musicbrainz.org/ws/2/url?query=url:*bing*search*&fmt=json

A more generic message would allow blocking pages from other popular search engines too:

```suggestion
        message: l(`Please don’t enter links to search results.
                    If you’ve found any links through your search
                    that seem useful, do enter those instead.`),
```

Reply:
Good point! I won't implement Bing blocking yet, but I will make the suggested change so the string doesn't have to change later :)